.circleci/deploy-testnet.sh

```diff
@@ -11,12 +11,36 @@ export COMMIT_HASH=$(./docker/hash.sh)
 
 cd .circleci && npm install @orbs-network/orbs-nebula && cd ..
 
-curl -O https://boyar-testnet-bootstrap.s3-us-west-2.amazonaws.com/boyar/config.json
-node .circleci/testnet-deploy-tag.js $COMMIT_HASH
-aws s3 cp --acl public-read config.json s3://boyar-testnet-bootstrap/boyar/config.json
+mkdir -p workspace
+echo "$TESTNET_NODE_IP" > workspace/testnet_ip
+
+if [ ! -z "$CI_PULL_REQUESTS" ]
+then
+    echo "We have an active PR ($CI_PULL_REQUESTS)"
```

Comment:
as discussed, in a (near) future PR you should set the vchain id from the PR/master here, and then proceed using a single flow